libpod/info.go

```diff
@@ -260,7 +261,14 @@ func (r *Runtime) storeInfo() (*define.StoreInfo, error) {
 			program["Version"] = ver
 			program["Package"] = version.Package(split[1])
 			graphOptions[split[0]] = program
-		} else {
+		case strings.HasSuffix(split[0], "imagestore"):
+			key := strings.ReplaceAll(split[0], "imagestore", "additionalImageStores")
```

Comment:
this means the old `imagestore` field is no longer included while the certainly was buggy it might be that users depend on that field somehow so we should not remove it and keep for backwards compat at least until 6.0 IMO

Reply:
Sure, I can keep that key as well, with the same logic if you think it can break someone.

Reply:
PR updated

Reply:
I think we need both.  imagestore refers to a path where images will be stored.  This feature is mostly used by CRI-O to split containers and images on two different file systems. It is not the same as additionalimagestores.

Reply:
@giuseppe I see that there are 2 different configs in `storage.conf`:

```toml
[storage]
imagestore = "/first/path"

[storage.options]
additionalimagestores = [ "second/path" ] 
```

But `podman info` returns only the `additionalimagestores` path:

```yaml
...
store:
  ...
  graphOptions:
    overlay.imagestore: "second/path"
    overlay.additionalImageStores: ["second/path"]
 ```

Reply:
CRI-O shouldn't be affected by what `podman info` returns (this PR change), right?